### PR TITLE
Support short flags for key, project, and config

### PR DESCRIPTION
**Pull request by Piccirello**

### CODE REVIEW

File: pkg/cmd/environments.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ var environmentsGetCmd = &cobra.Command{
 }
 
 func init() {
-	environmentsGetCmd.Flags().String("project", "", "output json")
+	environmentsGetCmd.Flags().StringP("project", "p", "", "output json")
```
Comment (bvallelunga) [inline pkg/cmd/environments.go]: shouldn't the description be "doppler project"?

File: pkg/cmd/configs.go [hunk 2]
```diff
@@ -192,46 +192,46 @@ var configsLogsRollbackCmd = &cobra.Command{
 }
 
 func init() {
-	configsCmd.Flags().String("project", "", "doppler project (e.g. backend)")
+	configsCmd.Flags().StringP("project", "p", "", "doppler project (e.g. backend)")
```
Comment (bvallelunga) [inline pkg/cmd/configs.go]: can we change all the "doppler project" to "enclave project"?

File: pkg/cmd/environments.go (hunk 1 above)
Comment (Piccirello) [reply pkg/cmd/environments.go]: good catch

File: pkg/cmd/root.go [hunk 3]
```diff
@@ -60,7 +60,7 @@ func init() {
 	rootCmd.Version = version.ProgramVersion
 	rootCmd.SetVersionTemplate(rootCmd.Version + "\n")
 
-	rootCmd.PersistentFlags().String("key", "", "doppler api key")
+	rootCmd.PersistentFlags().StringP("key", "k", "", "doppler api key")
```
Comment (bvallelunga) [inline pkg/cmd/root.go]: if we are moving to using the word token instead of key, should we switch the "k" to "t"?

Comment (Piccirello) [reply pkg/cmd/root.go]: Sure, I'll switch over to using `token` throughout. We'll have to manually update our existing config files. Alternatively, you can delete your config and let the migration script run again.

Comment (bvallelunga) [reply pkg/cmd/root.go]: Dope I will do that.

File: pkg/cmd/root.go [hunk 4]
```diff
@@ -60,12 +60,12 @@ func init() {
 	rootCmd.Version = version.ProgramVersion
 	rootCmd.SetVersionTemplate(rootCmd.Version + "\n")
 
-	rootCmd.PersistentFlags().String("key", "", "doppler api key")
+	rootCmd.PersistentFlags().StringP("token", "t", "", "doppler api key")
```
Comment (bvallelunga) [inline pkg/cmd/root.go]: Change "doppler api key" with "doppler token"

**Review by bvallelunga — APPROVED** [review]

**Pull request merged**